connectors/v2/genie.js

```diff
@@ -0,0 +1,19 @@
+'use strict';
+
+/* global Connector */
+
+Connector.playerSelector = '#music-player';
+Connector.artistSelector = '#ArtistNameArea';
+Connector.trackSelector = '#SongTitleArea';
+Connector.albumSelector = '#AlbumTilteArea';
+Connector.currentTimeSelector = '#playTime span';
+Connector.durationSelector = '#playTime strong';
+
+Connector.isPlaying = function() {
+	var btn = $('#PlayBtnArea')[0];
```

Comment:
I would simplify this to 

```
Connector.isPlaying = function () {
	return $('#PlayBtnArea').hasClass('active');
};
```

Or is there really more than one instance of `#PlayBtnArea`?